src/phoenix/trace/exporter.py

```diff
@@ -30,6 +32,80 @@ def export(self, _: Any) -> None:
         pass
 
 
+class OTELNoOpExporter(SpanExporter):
```

Comment:
SpanExporter is also defunct. Let's not inherit from it. In fact, this No Op Exporter is no longer necessary for OTEL because OTEL by default is no-op.

Reply:
addressed in 3d731601